docs/source/docs/get-started/quick-start-guide/installation.rst

```diff
@@ -28,7 +28,7 @@ Install:
 .. code-block::
 
     # From the GitHub repository:
-    pip install 'git+https://github.com/openvinotoolkit/datumaro[default]'
```

Comment:
Could you please elaborate this more?

This is because currently we have C++ API and Rust API. If we install from the source, C++ and Rust build systems should be prepared before. However, this is not pretty expected for the general users (especially for Rust). Therefore, I think that we can enhance this documentation such as

1. Recommend to install from the PyPI and explain why installation from the source is not recommended.
2. Describe the prerequisites of install from the source (How to prepare C++ and Rust build system)

Reply:
I added more comment to document. I think if the user can install datumaro from github, he/she can easily find how to prepare C++ and Rust build system. So I just added a note only.

Reply:
```text

1. From PyPI (**recommended**)

   .. code-block::

       pip install datumaro[default]

2. From source (**not recommended, for advanced users**)

   Installation from the repository source is not recommended.
   This is because it requires that C++ and Rust build systems are prepared in your local environment before installation.
   Datumaro includes C++ and Rust implementations to accelerate some workloads to overcome Python's innate slowness.

   .. code-block::

       # Prerequisite (For Unix-like systems)
       # Install C++ build system
       sudo apt-get install build-essential
       # Install Rust build system
       curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --help

       # Install from the GitHub repository
       pip install 'datumaro[default] @ git+https://github.com/openvinotoolkit/datumaro'

```

Reply:
Thank you. It looks much better.